src-java/flowhs-topology/flowhs-messaging/src/main/java/org/openkilda/messaging/command/flow/FlowRerouteFlushRequest.java

```diff
@@ -35,12 +35,16 @@ public class FlowRerouteFlushRequest extends CommandData {
     @JsonProperty("flow_id")
     private String flowId;
 
+    private String flowType;
```

Comment:
You can also use FlowType enum here, it might be better.

Reply:
fixed